org.gnomeradio.Radio.json

```
@@ -0,0 +1,52 @@
+{
+  "app-id" : "org.gnomeradio.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "45",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "gnome-radio",
+  "rename-icon" : "hicolor/scalable/apps/gnome-radio.svg",
+  "copy-icon" : true,
+  "rename-desktop-file" : "gnome-radio.desktop",
+  "copy-desktop-file" : true,
+  "rename-appdata-file" : "gnome-radio.appdata.xml",
+  "copy-appdata-file" : true,
+  "finish-args" : [
+     "--socket=wayland",
```

```suggestion
     "--socket=wayland",
     "--socket=fallback-x11",
```

You need this for users that can't run Wayland. Unless there is a good reason no to. (I doubt)